x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/allowed_values/helpers.test.tsx

```diff
@@ -79,60 +65,19 @@ describe('helpers', () => {
     });
 
     test('it returns an empty array when the `field` does NOT have `allowed_values`', () => {
-      expect(getValidValues(ecsMetadata['host.name'])).toEqual([]);
+      expect(getValidValues(EcsFlatTyped['host.name'])).toEqual([]);
     });
 
     test('it returns an empty array when `field` is undefined', () => {
       expect(getValidValues(undefined)).toEqual([]);
     });
-
-    test('it skips `allowed_values` where `name` is undefined', () => {
-      // omit the `name` property from the `database` `AllowedValue`:
-      const missingDatabase =
-        ecsMetadata['event.category'].allowed_values?.map((x) =>
-          x.name === 'database' ? omit<AllowedValue>('name', x) : x
-        ) ?? [];
-
-      const field = {
-        ...ecsMetadata['event.category'],
-        allowed_values: missingDatabase,
-      };
-
-      expect(getValidValues(field)).toEqual(
-        expect.arrayContaining([
-          'authentication',
-          'configuration',
-          'driver',
-          'email',
-          'file',
-          'host',
-          'iam',
-          'intrusion_detection',
-          'malware',
-          'network',
-          'package',
-          'process',
-          'registry',
-          'session',
-          'threat',
-          'vulnerability',
-          'web',
-        ])
-      );
-      expect(getValidValues(field)).not.toEqual(
-        expect.arrayContaining([
-          // there should be no entry for 'database'
-          'database',
-        ])
-      );
-    });
```

Comment:
For reviewers:

This test was removed because it tests an impossible state. Details:
https://github.com/elastic/kibana/pull/185025/files#r1631495151